students/RimleeT/lesson01/Break_me.py

```diff
@@ -0,0 +1,32 @@
+"""
+	In the break_me.py file write four simple Python functions:
+	Each function, when called, should cause an exception to happen
+	Each function should result in one of the four most common exceptions you’ll find.
+	for review: NameError, TypeError, SyntaxError, AttributeError
+"""
+
+#NameError 
+first_number = print("Input your first number")
+def name_error (first_number):	
+	print('first_number', fist_number)
```

Comment:
careful! these are <tab> characters for indents. It works, but can cause problems.

You want to use four spaces to indent with python. Always.

Make sure to set up your editor to put in four spaces when you hit the <tab> key.